homeassistant/components/alexa/handlers.py

```diff
@@ -924,3 +927,76 @@ def temperature_from_object(hass, temp_obj, interval=False):
     )
 
     return response
+
+
+@HANDLERS.register(("Alexa.ChannelController", "SkipChannels"))
+async def async_api_skip_channels(hass, config, directive, context):
+    """Process a skip channel request."""
+    # media_player channel up/down service does not support specifying steps
+    # each component handles it differently e.g. via config.
+    # For now we use the channelCount returned to figure out if we
+    # should step up/down
+    channel_count = directive.payload["channelCount"]
+    entity = directive.entity
+
+    data = {ATTR_ENTITY_ID: entity.entity_id}
+
+    if channel_count > 0:
+        await hass.services.async_call(
+            entity.domain,
+            SERVICE_MEDIA_NEXT_CHANNEL,
+            data,
+            blocking=False,
+            context=context,
+        )
+    elif channel_count < 0:
+        await hass.services.async_call(
+            entity.domain,
+            SERVICE_MEDIA_PREVIOUS_CHANNEL,
+            data,
+            blocking=False,
+            context=context,
+        )
+
+    return directive.response()
```

Comment:
```suggestion
    response = directive.response()
    response.add_context_property(
        {
            "namespace": "Alexa.ChannelController",
            "name": "channel",
            "value": {
              "number": "1234",
              "callSign": "callsign1",
              "affiliateCallSign": "callsign2"
            }
        )
        
    return response
```
According the the Alexa API:

> You must send an Response event if a request to change or skip channels was successful. The response should include a context object that reports the channel after the request completes.

Is there any way to report the current channel after the skip?

Reply:
My television (samsung) does not support detecting the current channel. Maybe other TV can, but I've no idea.
That's why I choose not to deal with the channel in response.
This can be added based on a new supported feature (linked to a new service) in the media player component.